back-end/src/sciler/config/configHandler.go

```diff
@@ -48,21 +48,17 @@ func generateDataStructures(readConfig ReadConfig) (WorkingConfig, []string) {
 	config.Puzzles = newPuzzles
 	newEvents, eventErrors := generateGeneralEvents(readConfig.GeneralEvents, &config)
 	config.GeneralEvents = newEvents
-	errorList = append(errorList, append(puzzleErrors, eventErrors...)...)
+	newButtonEvents, buttonEventErrors := generateButtonEvents(readConfig.ButtonEvents, &config)
+	config.ButtonEvents = newButtonEvents
+	errorList = append(errorList, append(append(puzzleErrors, eventErrors...), buttonEventErrors...)...)
```

Comment:
can you make it either append(append(append(puzzleErrors, eventErrors...), errorList...) , buttonEventErrors...)
or append(errorList, append(buttonEventErrors, append(puzzleErrors, eventErrors...)...)...)